core/src/test/java/org/apache/accumulo/core/file/blockfile/cache/TestLruBlockCache.java

```diff
@@ -243,9 +243,10 @@ public void testCacheEvictionTwoPriorities() throws Exception {
     BlockCacheManager manager = BlockCacheManagerFactory.getInstance(cc);
     cc.set(Property.TSERV_DEFAULT_BLOCKSIZE, Long.toString(blockSize));
     cc.set(Property.TSERV_INDEXCACHE_SIZE, Long.toString(maxSize));
-    LruBlockCacheConfiguration.builder(CacheType.INDEX).useEvictionThread(false).minFactor(0.98f)
-        .acceptableFactor(0.99f).singleFactor(0.25f).multiFactor(0.50f).memoryFactor(0.25f)
-        .buildMap().forEach(cc::set);
+    LruBlockCacheConfiguration.builder(CacheType.INDEX).useEvictionThread(false)
+        .minFactor(LruBlockCacheConfiguration.DEFAULT_ACCEPTABLE_FACTOR)
+        .acceptableFactor(0.99f).singleFactor(0.25f).multiFactor(0.50f)
+        .memoryFactor(0.25f).buildMap().forEach(cc::set);
```

Comment:
A `minFactor` was being set here to `0.98f`, however later in the test, it is performing an assertion based on the cache adhering to a `DEFAULT_ACCEPTABLE_FACTOR` (`0.85`).  I've changed the `minFactor` here to match the test and now it passes :)

Reply:
To be clear, the test was previously passing because It was inadvertently kicking out two blocks (should have only kicked out one) and therefore coming in below this lower value of `0.85`.  Now that it's configured to target this lower value, it is kicking out  two blocks (as expected) to meet this requirement.